lib/lacerda/compare/json_schema.rb

```diff
@@ -175,11 +169,11 @@ def schema_contains?(options)
         end
 
         if consume['type'] == 'array'
-          sorted_publish = publish['items'].sort

```

Comment:
I didn't understand what was going on here. To me `items` looks pretty much like a `oneOf`, you need to find at least one match. I had some issues with multitype arrays, that's why I re-did it

Reply:
You need to find one, but not more than one (otherwise it would be `anyOf`)